src/admin-views/editor/list-row.php

```diff
@@ -10,7 +10,7 @@
 $mode          = $ticket->global_stock_mode();
 $event         = $ticket->get_event();
 
-// If we don't have an event we should't even continue
+// If we don't have an event we shouldn't even continue
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: c5ac654e3ddba3504395f719f1d768d0</sub>